### PR TITLE
Updates to User Guide and API Docs

### PR DESCRIPTION
**Pull request by Alrobbertz**

### CODE REVIEW

**Comment by Alrobbertz** [comment]
I think this also fixes part of #20 but I'm not sure if it covers all of the bases.

**Comment by ehsteve** [comment]
Still working on making some changes to the plot function.

File: pyproject.toml [hunk 1]
```diff
@@ -10,7 +10,8 @@ description = "A Python package to support the HERMES instrument packages."
 readme = "README.rst"
 dynamic = ["version"]
 authors = [{name = "Steven Christe", email="steven.d.christe@nasa.gov"},
-           {name = "Damian Barrous Dumme", email="damianbarrous@gmail.com"}]
+           {name = "Damian Barrous Dumme", email="damianbarrous@gmail.com"},
+           {name = "Andrew Robbertz", email=""]
```
Comment (Alrobbertz) [inline pyproject.toml:14]:
```suggestion
           {name = "Andrew Robbertz", email="a.robbertz@gmail.com"]
```

**Comment by ehsteve** [comment]
@Alrobbertz I'm done with my changes now.

**Comment by ehsteve** [comment]
When you're ready we should make a merge request into main which will run the tests. Would be good to get someone to review the guide on that merge request.

File: hermes_core/timedata.py [hunk 2]
```diff
@@ -232,27 +223,44 @@ def __iter__(self):
...
+        that must be set for a valid CDF file to be generated from the `TimeData` container.
+
+        Parameters
+        ----------
+        instr_name : str
+            The instrument name. Must be "eea", "nemisis", "merit" or "spani".
+        data_level : str
+            The data level of the data. Must be "l1", "ql", "l3", "l4"
+        version : str
+            Must be of the form X.Y.Z.
```
[the remark this answers is not in the document]
Comment (ehsteve) [reply hermes_core/timedata.py:235]: This is a wrapper to the schéma function. If users want an empty version they could just use that function right?

[the remark this answers is not in the document]
Comment (Alrobbertz) [reply hermes_core/timedata.py:235]: Yes, but I already updated to make them optional arguments and some validation that the strings match the format expected.

**Comment by Alrobbertz** [comment]
> When you're ready we should make a merge request into main which will run the tests. Would be good to get someone to review the guide on that merge request.

Do you want me to update some test cases as a part of this PR or would you like me to add test cases directly to the `cdf_data` branch when we're merging it to main?

**Pull request merged**